modules/rostests/winetests/kernel32/actctx.c

```diff
@@ -3834,7 +3841,13 @@ static void test_builtin_sxs(void)
     ULONG_PTR cookie;
     HANDLE handle_context;
     BOOL success;
+#ifdef __REACTOS__
+    static char expected_path[MAX_PATH];
+    GetWindowsDirectoryA(expected_path, sizeof(expected_path));
+    strcat(expected_path, "\\WinSxS");
```

Comment:
maybe do (since the buffer is "static"):
```suggestion
    static char expected_path[MAX_PATH] = {0};
    if (!*expected_path)
    {
        GetWindowsDirectoryA(expected_path, sizeof(expected_path));
        strcat(expected_path, "\\WinSxS");
    }
```